workflows/iuf/operations/preflight-checks-for-services.yaml

```diff
@@ -44,13 +44,6 @@ spec:
             templateRef: 
               name: iuf-base-template
               template: shell-script
-            retryStrategy:
```

Comment:
if you want, you can move this up under spec but i see you are using the same as defined in top level workflow so it is not necessary. but if you need special handling, that's how you can override that